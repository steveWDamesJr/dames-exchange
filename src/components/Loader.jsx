import React from 'react';

function Loader() {
  return (
    <div>Loader</div>
  );
}

export default Loader;
