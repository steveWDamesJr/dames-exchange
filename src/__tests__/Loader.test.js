import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../components/Loader';

it('renders correctly', () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
