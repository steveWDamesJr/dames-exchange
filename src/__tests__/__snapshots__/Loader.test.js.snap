// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loader"
>
  <div
    aria-busy={true}
    aria-live="polite"
    className="ant-spin ant-spin-spinning ant-spin-show-text"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
    <div
      className="ant-spin-text"
    >
      Dames Exchange Loading...
    </div>
  </div>
</div>
`;
